src/Yasumi/Provider/USA.php

```diff
@@ -161,9 +161,18 @@ private function calculateMemorialDay(): void
     private function calculateJuneteenth(): void
     {
         if ($this->year >= 2021) {
+            $date = new DateTime("$this->year-6-19", DateTimeZoneFactory::getDateTimeZone($this->timezone));
+            $label = 'Juneteenth';
+            if ('Sunday' === $date->format('l')) {
```

Comment:
I'd recommend to use the 'w' formatter (A numeric representation of the day (0 for Sunday, 6 for Saturday)) instead, as the string representation of the week day is less certain to be stable (may have future changes in casing, naming, etc.)